app/docs/page.jsx

```
@@ -0,0 +1,89 @@
+        <section className="space-y-6">
+          <h2 className="flex items-center gap-2 text-2xl font-semibold text-vault-text">
+            <RefreshCw className="text-vault-accent" /> Round Lifecycle
+          </h2>
+          <div className="prose prose-invert max-w-none text-vault-muted">
+            <p>
+              The protocol operates in <strong>Rounds</strong>. At the end of each round:
+            </p>
+            <ol className="list-decimal pl-6 space-y-2">
+              <li>The total yield generated across all strategies is calculated.</li>
+              <li>A verifiable random function (VRF) is used to select winners based on their deposit size and duration (time-weighted balance).</li>
+              <li>Prizes are distributed to the winners' wallets or added to their vault balances.</li>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)